contract-rs/01-basic-auction/Cargo.toml

```diff
@@ -9,10 +9,10 @@ crate-type = ["cdylib", "rlib"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-near-sdk = { version = "5.3" }
+near-sdk = { version = "5.4.0" }
 
 [dev-dependencies]
-near-sdk = { version = "5.3", features = ["unit-testing"] }
+near-sdk = { version = "5.4.0", features = ["unit-testing"] }
 near-workspaces = { version = "0.11.0", features = ["unstable"] }
```

Comment:
The latest near-workspaces version is 0.14.0. Could you try to update it in dependencies?